packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -49,6 +51,18 @@ function addVNodeToChildLWC(vnode: VCustomElement) {
     ArrayPush.call(getVMBeingRendered()!.velements, vnode);
 }
 
+// [st]atic node
+function st(elmProto: Element, key: Key): VStatic {
+    return {
+        type: VNodeType.Static,
+        sel: undefined,
+        key,
+        elm: undefined,
+        elmProto,
```

Comment:
this property should probably be renamed to `frag` or `fragment`, using `Proto` postfix is misleading.